requirements.txt

```diff
@@ -1,10 +1,16 @@
 numpy>=1.17.0
 pandas>=0.25.0
-gym>=0.14.0
+gym==0.15.7
+ray[default,tune,rllib,serve]==1.9.2
 pyyaml>=5.1.2
 stochastic==0.6.0
-tensorflow>=2.1.0
+tensorflow==2.4.2
```

Comment:
Why pinning a specific TF version? Did you have any issues with more recent versions?
I am currently using 2.7.0 without specific issues.

Reply:
The Dockerfile was 2.1.0 and the tensorflow version (2.6) was then redownloaded, overwriting and breaking tensorflow-estimator. Without pinning, it would reinstall. Tensorflow 2.4.2 is the latest version I have tried while using tensortrade.  I have yet to try with tensorflow 2.7.0. This environment just works for now until I try it with tensorflow 2.7.0

Reply:
updated to tensorflow 2.7.0